docs/docs/core-concepts/caching.md

```diff
@@ -1,47 +1,70 @@
+the actual memory usage of your process. You'll need to fine tune it to your
+actual requirements. As an example if you have 1 million states, each of which
+is 50 bytes that's 50MB of memory.
 
-On top of the cache we have a few Caching strategies to choose from
+The cache acts as a concurrency limiter on stream loads and guarantees that
+for each stream at most one load is happening concurrently.
 
-# NoCaching
+For performance reasons we do not offer TTL functionality on the cache. Doing
+so with the guarantees we'd want is not feasible with the underlying cache
+library. In particular, the underlying library will not pre-emptively evict
+expired items from the cache and as such your items would remain in the cache
+until requested again, at which point they'd be evicted. As such caching in
+EquinoxJS is a binary on/off setting.
+
+## `NoCaching`
 
 ```ts
 import { CachingStrategy } from "@equinox-js/core"
-const strategy = CachingStrategy.noCache()
+const strategy = CachingStrategy.NoCache()
 ```
 
 No caching is applied. This necessitates a fresh load of the entire stream for every interaction with the
 store.
 
-# SlidingWindow(cache, ms, ?prefix)
+## `Cache(cache, ?prefix)`
 
 ```ts
 import { CachingStrategy, MemoryCache } from "@equinox-js/core"
 const cache = new MemoryCache()
-const strategy = CachingStrategy.slidingWindow(cache, 20 * 60 * 1000)
+const strategy = CachingStrategy.Cache(cache)
 ```
 
-Retain a single `state` per `stream_name`.
-Each cache hit for a stream renews the retention period for the defined `ms`.
-Upon expiration of the defined `ms` from the point at which the cache was entry was last used, a full reload is
-triggered.
-Unless `LoadOption.AllowStale` is used, each cache hit still incurs a roundtrip to load any subsequently-added events.
+Retain a single `state` per `stream_name` in the supplied cache.
 
-Prefix is used to segregate multiple folds per stream when they are stored in the cache. 
+Prefix is used to segregate multiple folds per stream when they are stored in the cache.
 
-# FixedTimeSpan(cache, periodMs)
+# LoadOptions
 
-```ts
-import { CachingStrategy, MemoryCache } from "@equinox-js/core"
-const cache = new MemoryCache()
-const strategy = CachingStrategy.fixedTimeSpan(cache, 20 * 60 * 1000)
-```
+When transacting or querying on your decider you can supply a load option to
+opt into different load guarantees.
+
+## `RequireLoad`
+
+This is the default load option, always obtaining the latest state from the store
+
+## `RequireLader`
+
+Request that the stream be loaded with a quorum read / from a leader connection
+
+## MaxStale(ms)
+
+If the cache contains a state from the stream that was stored less than `ms`
+milliseconds ago that state will be used without fetching more events from the
+stream.
+
+## `AnyCachedValue`
+
+If the cache contains any state from the stream, that state will be used without
+fetching more events from the stream.
```

Comment:
Useful to avoid a read roundtrip if you believe there is a good chance that the value in the cache will be correct (if not, and append will trigger a reload due to a concurrency conflict during the <sync operation, whatever you term it in here>)